packages/cli/src/lib/cmds/help.ts

```diff
@@ -84,10 +86,27 @@ const HELP_MESSAGES = new Map<string, string>(
             '"./app-release-signed.apk"',
         '--verbose ................. prints the adb command being executed',
       ].join('\n')],
+      ['doctor', [
+        'Usage:',
+        '',
+        '',
+        'bubblewrap doctor',
+      ].join('\n')],
+      ['updateConfig', [
+        'Usage:',
+        '',
+        '',
+        'bubblewrap updateConfig --<jdk or androidSdk> new/path/to/folder',
```

Comment:
Can I invoke the command with both --jdk and --androidSdk? Maybe something like this would work better:
```suggestion
        'bubblewrap updateConfig [--jdk <path-to-jdk>] [--androidSdk <path-to-android-sdk>]',
```

Also, we probably want to add a note that at least one option is required.